Swift/ChartboostExample/AppDelegate.swift

```diff
@@ -15,6 +16,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         print("Chartboost SDK Version ", Chartboost.getSDKVersion() ?? "")
+        
+        if #available(iOS 14, *) {
+            ATTrackingManager.requestTrackingAuthorization { status in
+                print("Tracking authorization status changed: \(status)")
```

Comment:
should we print `status.rawValue`? I think otherwise we just get a `ATTrackingManagerAuthorizationStatus` string no matter the case

Reply:
good catch! I thought I read "Authorized" instead of "Authorization" when testing it